src/Native/Unix/System.Security.Cryptography.Native/opensslshim.h

```diff
@@ -1007,6 +1007,12 @@ FOR_ALL_OPENSSL_FUNCTIONS
 #define X509_get_version local_X509_get_version
 #define X509_up_ref local_X509_up_ref
 
+#if OPENSSL_VERSION_NUMBER < OPENSSL_VERSION_1_0_2_RTM
+
+#define X509_STORE_CTX_get0_store local_X509_STORE_CTX_get0_store
```

Comment:
Would it make sense to just do it as ifndef then define, rather than basing it on version?

Reply:
`#ifndef` only looks at `#define` tokens, not function names, so `#ifndef X509_STORE_CTX_get0_store` is always true.

This `#if` matches the one for the portable version which defined the missing function prototype; I just missed this symmetry case.